src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ProjectItemsSchema.xaml

```diff
@@ -58,9 +58,10 @@
       <NameValuePair Name="DefaultMetadata_Generator" Value="ResXFileCodeGenerator" />
     </ContentType>
 
+    <ItemType Name="AdditionalFiles" DisplayName="AdditionalFiles"/>
```

Comment:
Can we give this a better display name? The others don't use compound pascal cased, they use sentence case. Also can we give a name that indicates it gets passed the compiler for an analyzer? Additional files is just too generate despite that's what we called the item.